# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wmx/compilefix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ruN wmx-6/Menu.C wmx-6-avm/Menu.C
--- wmx-6/Menu.C	2000-05-24 17:50:27.000000000 +0200
+++ wmx-6-avm/Menu.C	2003-06-13 23:17:17.000000000 +0200
@@ -621,7 +621,7 @@
 
 
 CommandMenu::CommandMenu(WindowManager *manager, XEvent *e,
-			 char* otherdir = NULL)
+			 char* otherdir)
     : Menu(manager, e)
 {
     const char *home = getenv("HOME");
